<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DestinationsPageWidget</class>
 <widget class="QWidget" name="DestinationsPageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1334</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1322</width>
        <height>250</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QPushButton" name="btnDestAdd">
           <property name="text">
            <string>Добавить..</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDestAddToComplex">
           <property name="text">
            <string>Добавить в комплекс</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDestSet">
           <property name="text">
            <string>Назначить</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDestSave">
           <property name="text">
            <string>Сохранить</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDestDelete">
           <property name="text">
            <string>Удалить</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDestCancel">
           <property name="text">
            <string>Отменить</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDestFind">
           <property name="text">
            <string>Найти</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="CPrintButton" name="btnDestPrint">
           <property name="text">
            <string>Печать</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_17">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnDestAll">
           <property name="text">
            <string>Показать все назначения</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="CDestinationsTableView" name="tblDestinations">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>200</height>
          </size>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1" rowspan="2">
        <widget class="QGroupBox" name="grpDestFilter">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>Фильтр</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_28">
          <item row="9" column="2">
           <widget class="CApplyResetDialogButtonBox" name="bbxDestFilter">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="standardButtons">
             <set>QDialogButtonBox::Apply|QDialogButtonBox::Reset</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>по</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>В период с</string>
            </property>
           </widget>
          </item>
          <item row="8" column="2">
           <spacer name="verticalSpacer_7">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>316</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="9" column="0">
           <spacer name="horizontalSpacer_19">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="lblDestFilterStatus">
            <property name="text">
             <string>Статус</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="lblDestFilterDrug">
            <property name="text">
             <string>Препарат</string>
            </property>
           </widget>
          </item>
          <item row="6" column="2" colspan="2">
           <widget class="QComboBox" name="cmbDestFilterStatus">
            <property name="minimumSize">
             <size>
              <width>170</width>
              <height>0</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>все</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>новый</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>создан</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>назначен</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>отменён</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>выполнен</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="5" column="2" colspan="2">
           <widget class="QLineEdit" name="edtDestFilterDrug">
            <property name="minimumSize">
             <size>
              <width>170</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="2" colspan="2">
           <widget class="QDateEdit" name="edtDestFilterEndDate">
            <property name="minimumSize">
             <size>
              <width>170</width>
              <height>0</height>
             </size>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2" colspan="2">
           <widget class="QDateEdit" name="edtDestFilterBegDate">
            <property name="minimumSize">
             <size>
              <width>170</width>
              <height>0</height>
             </size>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CPrintButton</class>
   <extends>QPushButton</extends>
   <header>library/PrintTemplates.h</header>
  </customwidget>
  <customwidget>
   <class>CApplyResetDialogButtonBox</class>
   <extends>QDialogButtonBox</extends>
   <header>library/DialogButtonBox.h</header>
  </customwidget>
  <customwidget>
   <class>CDestinationsTableView</class>
   <extends>QTableView</extends>
   <header>Events/DestinationsTable.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
